src/closure-externs.js

```diff
@@ -763,6 +763,7 @@ SIMD.Bool64x2.fromFloat64x2 = function() {};
 
 /**
  * @const
+ * @suppress {duplicate}
```

Comment:
Can you `ifndef WASMJS` to avoid the duplicate?

Reply:
I don't think we can do that in these closure files - we don't run the preprocessor on them.